TOMP-API.yaml

```diff
@@ -658,6 +657,29 @@ paths:
         '404':
           $ref: '#/components/responses/404NotFound'
 
+  /operator/meta:
+    parameters:
+      - $ref: '#/components/parameters/acceptLanguage'
+    get:
+      summary: describes the running implementations
+      description: all versions that are implemented on this url, are described in the result of this endpoint. In contains all versions and per version the endpoints, their status
+        and the supported scenarios.
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
```

Comment:
Why is this an array? endpointImplementation describes one operator right?

Reply:
nope. An operator can serve multiple versions. maybe 'endpointVersion' should be a better word.